whois-commons/src/main/java/net/ripe/db/whois/common/DateTimeProvider.java

```diff
@@ -13,8 +13,8 @@
     LocalDateTime getCurrentDateTime();
     ZonedDateTime getCurrentDateTimeUtc();
 
-    /** returns System.nanoTime(), the high-res timer that counts 0 from JVM startup */
-    long getNanoTime();
+    /** returns elapsed time. N.B. not related to system time */
+    long getElapsedTime();
```

Comment:
Maybe call it getElapsedTimeNanos() so it's clear that the unit is nanons and not milliseconds..?

Reply:
I'd prefer not to include the unit precision in the method name, it makes it very verbose. I updated the comment to be more specific.